fhir-server/src/main/java/com/ibm/fhir/server/resources/WellKnown.java

```diff
@@ -99,41 +107,36 @@ private JsonObject buildSmartConfig() throws Exception {
         JsonObjectBuilder responseBuilder = Json.createObjectBuilder();
 
         if (regURL != null && !regURL.isEmpty()) {
-            responseBuilder.add("registration_endpoint", regURL);
+            responseBuilder.add("registration_endpoint", regURL); // recommended
         }
 
-        return responseBuilder
-                .add("authorization_endpoint", authURL) // required
-                .add("token_endpoint", tokenURL) // required
-                .add("scopes_supported", Json.createArrayBuilder() // recommended
-                    .add("openid")
-                    .add("profile")
-                    .add("offline_access")
-                    // TODO
-//                    .add("launch")
-//                    .add("launch/patient")
-//                    .add("user/*.*")
-//                    .add("patient/*.*")
-                    .build())
-                .add("response_types", Json.createArrayBuilder() // recommended
-                    .add("code")
-                    .add("token")
-                    .add("id_token token")
-                    .build())
-                .add("capabilities", Json.createArrayBuilder() // required
-                    .add("launch-standalone")
-                    .add("client-public")
-                    .add("client-confidential-symmetric")
-                    .add("permission-offline")
-                    // TODO
-//                    .add("context-standalone-patient")
-//                    .add("context-standalone-encounter")
-//                    .add("permission-user")
-//                    .add("permission-patient")
-                    .build())
+        responseBuilder.add("authorization_endpoint", authURL); // required
+        responseBuilder.add("token_endpoint", tokenURL); // required
+
+        if (supportedScopes != null && !supportedScopes.isEmpty()) {
+            responseBuilder.add("scopes_supported", Json.createArrayBuilder(supportedScopes).build()); // recommended
+        }
+
+        responseBuilder.add("response_types", Json.createArrayBuilder() // recommended
+                .add("code")
+                .add("token")
+                .add("id_token token")
+                .build());
+        responseBuilder.add("capabilities", Json.createArrayBuilder() // required
+                .add("launch-standalone")
+                .add("client-public")
+                .add("client-confidential-symmetric")
+                .add("permission-offline")
+                // TODO: make this configurable
+                .add("context-standalone-patient")
+                .add("permission-patient")
+//                .add("context-standalone-encounter")
+//                .add("permission-user")
```

Comment:
does this deserve a backlog issue?

Reply:
I could maybe just "fix" it by adding another property to the config.  let me just do that now.

Reply:
no worries, just trying to take a critical eye to any change

Reply:
done